packages/api/tests/lib/rules/test-rulesHelpers.js

```diff
@@ -1712,8 +1705,7 @@ test.serial('Creating an enabled SNS rule creates an event source mapping', asyn
     state: 'ENABLED',
   });
   const subscribeSpy = sinon.spy(awsServices.sns(), 'subscribe');
-  const addPermissionSpy = sinon.spy(awsServices.lambda(), 'addPermission');
-
+  const lambdaMock = sinon.spy(awsServices.lambda(), 'send');
```

Comment:
Hmm we might want to be more specific here and listen for `addPermission`?

Reply:
Fair point, will address

Reply:
91a96be9c4b